helm/cluster-azure/values.schema.json

```diff
@@ -1,135 +1,180 @@
 {
     "$schema": "http://json-schema.org/schema#",
+    "description": "Configuration of an Azure cluster using Cluster API",
     "properties": {
         "attachCapzControllerIdentity": {
+            "title": "Attach controller identity",
             "type": "boolean"
         },
         "azure": {
             "properties": {
                 "azureClusterIdentity": {
                     "properties": {
                         "name": {
+                            "title": "Name",
                             "type": "string"
                         },
                         "namespace": {
+                            "title": "Namespace",
                             "type": "string"
                         }
                     },
+                    "title": "Cluster identity",
```

Comment:
i would change this to `AzureClusterIdentity used to create the Workload Cluster`

Reply:
Is `AzureClusterIdentity` a technical term that we expect users to know? Where is is defined? (GS, Azure)

Reply:
OK, found it documented in https://capz.sigs.k8s.io/topics/multitenancy.html . In that case I would suggest to add a description and use the term in that one.